joomla/joomla-cms
# [4.0] Fix: Saving menu item fails. Call to a member function validate() on bool. Field "show_associations"

Pull Request for Issue #23696

Related and already merged fix: #23501

I think this is a better alternative fix to #23700 ?


## Testing Instructions
Install a J4 nightly build of today (Monday, 28 January 2019) or update an older J4 with that package

Go to menu items manager in back-end.

Open Home menu item (type "Articles > Featured Articles")

Save it.

Error like described in #23696 (comment)

Apply patch

Try again to save the menu item.

Should be successful.
